api/internal/localizer/util.go

```diff
@@ -99,3 +101,7 @@ func hasRef(repoURL string) bool {
 	}
 	return repoSpec.Ref != ""
 }
+
+func isBuiltinPlugin(res *resource.Resource) bool {
```

Comment:
This function originated from https://github.com/kubernetes-sigs/kustomize/blob/master/api/internal/plugins/loader/loader.go#L177

I can just make it public? Asking because we agreed to limit the scope of my `localize` code changes.

Reply:
I've moved `isBuiltinPlugin` from `api/internal/plugins/loader` to `api/internal/plugins/utils`, made it public, and added unit tests. This line is now calling `utils.IsBuiltinPlugin()`.

Note that the unit tests I've added reflect the existing implementation of `isBuiltinPlugin`. I'm pretty sure 2 of them are wrong, as I've commented inline. Let me know if you want me to address them.

Reply:
EDIT: I adjusted the scope of this PR. This function no longer exists.